libraries/delivery_supermarket.rb

```diff
@@ -69,7 +70,7 @@ def remove_supermarket_key
 
     # Share the cookbook to the Supermarket Site
     def share_cookbook_to_supermarket(options)
-      command = "knife supermarket share #{new_resource.cookbook} " \
+      command = "knife supermarket share #{new_resource.cookbook} #{new_resource.category}" \
```

Comment:
I think this field should be surrounded by double quotes: 
```
$ knife supermarket share my_apache2_cookbook "Web Servers"
```
Because there are some categories that has spaces: https://docs.chef.io/plugin_knife_supermarket.html#share